adapters/raw/raw.go

```diff
@@ -16,6 +17,17 @@ func init() {
 	router.AdapterFactories.Register(NewRawAdapter, "raw")
 }
 
+var funcs = template.FuncMap{
+	"toJSON": func(value interface{}) string {
+		bytes, err := json.Marshal(value)
+		if err != nil {
+			log.Println("raw:", err)
```

Comment:
Can we preface the error with something more descriptive than `raw:`? `error` perhaps?

Reply:
I used the same from here: https://github.com/lucassabreu/logspout/blob/a627bc5fbc5106d39cb7bf73de83ba016aee8b9c/adapters/raw/raw.go#L75

but looking now, it makes more sense to have more context.